backend/src/routes/groups-routes.ts

```diff
@@ -0,0 +1,16 @@
+import express from 'express';
+import {createGroup, retrieveGroups, getSpecificGroup, updateName, deleteGroup} from '../controllers/groups-controller';
+import {body} from "express-validator";
+
+const router = express.Router();
+
+router.post('/', [
+    body('name').isString().withMessage('Name must be a string').notEmpty().withMessage('Name is required').isLength({ min: 3, max: 25 })], createGroup);
+router.get('/', retrieveGroups);
+router.get('/',[
```

Comment:
Remove the Express Validator middleware here. `GET` requests can't have a Body, so there's no need to use express-validator here